.github/workflows/terraform-apply.yml

```
@@ -11,13 +11,13 @@ on:
 permissions:
   contents: read
   id-token: write
-  pull-requests: write
+  pull-request: write
```

actionlint (1.7.9)

unknown permission scope "pull-request". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)